sphinx/make.bat

```diff
@@ -34,11 +34,12 @@ if "%1" == "clean" (
 
 if "%1" == "all" (
     make html
+	make serve
```

Comment:
nit: indentation

Reply:
Same thing here, this should never have been part of this commit and was included accidentally.